habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/BuildDeploymentArtifactsMojo.java

```diff
@@ -26,68 +28,94 @@
 public class BuildDeploymentArtifactsMojo extends AbstractHabushuMojo {
 
     /**
-     * By default, export requirements.txt file
+     * By default, export requirements.txt file.
      */
     @Parameter(property = "habushu.exportRequirementsFile", required = false, defaultValue = "true")
     protected boolean exportRequirementsFile;
 
     /**
-     * By default, do not include the --without-urls flag when exporting
+     * By default, do not include the --without-urls flag when exporting.
      */
     @Parameter(property = "habushu.exportRequirementsWithUrls", required = false, defaultValue = "true")
     protected boolean exportRequirementsWithUrls;
 
     /**
-     * By default, do not include the --without-hashes flag when exporting
+     * By default, do not include the --without-hashes flag when exporting.
      */
     @Parameter(property = "habushu.exportRequirementsWithHashes", required = false, defaultValue = "true")
     protected boolean exportRequirementsWithHashes;
 
     /**
-     * By default, export to the dist folder to be included with the build archive
+     * By default, export to the dist folder to be included with the build archive.
      */
     @Parameter(property = "habushu.exportRequirementsFolder", required = false, defaultValue = "${project.basedir}/dist")
     protected String exportRequirementsFolder;
 
+    /**
+     * Location of the artifact that will be published for this module.
+     */
+    @Parameter(property = "habushu.mavenArtifactFile", required = true, defaultValue = "${project.basedir}/target/habushu.placeholder.txt")
```

Comment:
Should you also include instructions in the readme about this new parameter?

Reply:
Yeah - I was thinking that this was more of a hidden param that _could_ be changed, but no harm in exposing it more explicitly.